tests/test_model.py

```diff
@@ -5833,19 +5833,43 @@ def test_lookup(self):
         source = DataSource.lookup(self._db, DataSource.OCLC)
```

Comment:
NOTE: If there is no collection on a CoverageRecord, you can look it up by passing in collection=None but not by passing in some other thing.